ui/app/components/app-form/project-settings.ts

```diff
@@ -48,24 +48,24 @@ const DEFAULT_PROJECT_MODEL = {
     git: {
       url: '',
       path: '',
-      ref: 'HEAD',
+      ref: 'MAIN',
       basic: {
         username: '',
-        password: ''
+        password: '',
       },
       ssh: {
         user: '',
         password: '',
         privateKeyPem: '',
-      }
+      },
```

Comment:
is this comma needed here?

Reply:
hmm, i also see a trailing comma on line 59 and 54 too. is that the code convention?

Reply:
Yes, the linter enforces a `comma on last line of objects` rule